### DISCUSSION TITLE
Which module is Konsist meant to run on?

### DISCUSSION THREAD

**Discussion by eygraber** [LemonAppDev/konsist#875] [Ask a Question]
### Wanted end result.

Which module is Konsist meant to run on? Is it supposed to be written on a per module basis, or only on the "app" module?

### What you tried so far.

It's not clear from the documentation what the intention is.

**Comment by igorwojda** [comment]
It is us to dev to decide, but my suggestion si to keep all Konsist tests in a single place (as they affect entire codebase). You can use Dedicated Source Set or Dedicated Module 

edit (upd link)
https://docs.konsist.lemonappdev.com/advanced/isolate-konsist-tests
